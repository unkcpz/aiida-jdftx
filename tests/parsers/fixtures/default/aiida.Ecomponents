   Eewald =       -8.3994724711985196
       EH =        0.5503370049656557
     Eloc =       -2.5565693402334086
      Enl =        1.8451129095899654
      Exc =       -2.4078586928891821
       KE =        3.0855137196355500
-------------------------------------
     Etot =       -7.8829368701299387
