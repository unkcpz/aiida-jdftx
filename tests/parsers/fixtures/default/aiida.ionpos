# Ionic positions in cartesian coordinates:
ion Si   0.000000000000000   0.000000000000000   0.000000000000000 0
ion Si   2.565303029500000   2.565303029500000   2.565303029500000 0

