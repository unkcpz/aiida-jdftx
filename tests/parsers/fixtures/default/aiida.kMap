#ReducedKpt #Symmetry inversion      (0-based indices, unreduced k-points in C array order)
 0  0 +1
 1  0 +1
 2  0 +1
 3  0 +1
 4  0 +1
 5  0 +1
 6  0 +1
 7  0 +1
 1 29 +1
 8  0 +1
 9  0 +1
10  0 +1
11  0 +1
12  0 +1
13  0 +1
14  0 +1
 2 29 +1
 9 39 +1
15  0 +1
16  0 +1
17  0 +1
18  0 +1
19  0 +1
20  0 +1
 3 29 +1
10 39 +1
16 39 +1
21  0 +1
22  0 +1
23  0 +1
24  0 +1
25  0 +1
 4 29 +1
11 39 +1
17 39 +1
22 39 +1
26  0 +1
27  0 +1
28  0 +1
25 20 +1
 5 29 +1
12 39 +1
18 39 +1
23 39 +1
27 39 +1
29  0 +1
30  0 +1
20 20 +1
 6 29 +1
13 39 +1
19 39 +1
24 39 +1
28 39 +1
30 39 +1
31  0 +1
14 20 +1
 7 29 +1
14 39 +1
20 39 +1
25 39 +1
25 14 +1
20 14 +1
14 14 +1
 7 14 +1
 1 27 +1
 8 33 +1
 9 35 +1
10 35 +1
11 35 +1
12 35 +1
13 35 +1
14 35 +1
 8 27 +1
32  0 +1
33  0 +1
34  0 +1
35  0 +1
36  0 +1
37  0 +1
31 18 +1
 9 29 +1
33 29 +1
38  0 +1
39  0 +1
40  0 +1
41  0 +1
42  0 +1
30 20 +1
10 29 +1
34 29 +1
39 39 +1
43  0 +1
44  0 +1
45  0 +1
46  0 +1
28 20 +1
11 29 +1
35 29 +1
40 39 +1
44 39 +1
47  0 +1
48  0 +1
46 20 +1
24 20 +1
12 29 +1
36 29 +1
41 39 +1
45 39 +1
48 39 +1
49  0 +1
42 20 +1
19 20 +1
13 29 +1
37 29 +1
42 39 +1
46 39 +1
46 14 +1
42 14 +1
37 14 +1
13 20 +1
14 29 +1
31 12 +1
30 14 +1
28 14 +1
24 14 +1
19 14 +1
13 14 +1
 6 14 +1
 2 27 +1
 9 33 +1
15 33 +1
16 35 +1
17 35 +1
18 35 +1
19 35 +1
20 35 +1
 9 27 +1
33 27 +1
38 33 +1
39 35 +1
40 35 +1
41 35 +1
42 35 +1
30 18 +1
15 27 +1
38 27 +1
50  0 +1
51  0 +1
52  0 +1
53  0 +1
49 18 +1
29 18 +1
16 29 +1
39 29 +1
51 29 +1
54  0 +1
55  0 +1
56  0 +1
48 20 +1
27 20 +1
17 29 +1
40 29 +1
52 29 +1
55 39 +1
57  0 +1
56 20 +1
45 20 +1
23 20 +1
18 29 +1
41 29 +1
53 29 +1
56 39 +1
56 14 +1
53 14 +1
41 20 +1
18 20 +1
19 29 +1
42 29 +1
49 12 +1
48 14 +1
45 14 +1
41 14 +1
36 14 +1
12 20 +1
20 29 +1
30 12 +1
29 12 +1
27 14 +1
23 14 +1
18 14 +1
12 14 +1
 5 14 +1
 3 27 +1
10 33 +1
16 33 +1
21 33 +1
22 35 +1
23 35 +1
24 35 +1
25 35 +1
10 27 +1
34 27 +1
39 33 +1
43 33 +1
44 35 +1
45 35 +1
46 35 +1
28 18 +1
16 27 +1
39 27 +1
51 27 +1
54 33 +1
55 35 +1
56 35 +1
48 18 +1
27 18 +1
21 27 +1
43 27 +1
54 27 +1
58  0 +1
59  0 +1
57 18 +1
47 18 +1
26 18 +1
22 29 +1
44 29 +1
55 29 +1
59 29 +1
59 14 +1
55 20 +1
44 20 +1
22 20 +1
23 29 +1
45 29 +1
56 29 +1
57 12 +1
55 14 +1
52 14 +1
40 20 +1
17 20 +1
24 29 +1
46 29 +1
48 12 +1
47 12 +1
44 14 +1
40 14 +1
35 14 +1
11 20 +1
25 29 +1
28 12 +1
27 12 +1
26 12 +1
22 14 +1
17 14 +1
11 14 +1
 4 14 +1
 4 27 +1
11 33 +1
17 33 +1
22 33 +1
26 33 +1
27 35 +1
28 35 +1
25 18 +1
11 27 +1
35 27 +1
40 33 +1
44 33 +1
47 33 +1
48 35 +1
46 18 +1
24 18 +1
17 27 +1
40 27 +1
52 27 +1
55 33 +1
57 33 +1
56 18 +1
45 18 +1
23 18 +1
22 27 +1
44 27 +1
55 27 +1
59 27 +1
59  8 +1
55 18 +1
44 18 +1
22 18 +1
26 27 +1
47 27 +1
57 27 +1
59  6 +1
58  6 +1
54 18 +1
43 18 +1
21 18 +1
27 29 +1
48 29 +1
56 12 +1
55 12 +1
54 12 +1
51 14 +1
39 20 +1
16 20 +1
28 29 +1
46 12 +1
45 12 +1
44 12 +1
43 12 +1
39 14 +1
34 14 +1
10 20 +1
25 12 +1
24 12 +1
23 12 +1
22 12 +1
21 12 +1
16 14 +1
10 14 +1
 3 14 +1
 5 27 +1
12 33 +1
18 33 +1
23 33 +1
27 33 +1
29 33 +1
30 35 +1
20 18 +1
12 27 +1
36 27 +1
41 33 +1
45 33 +1
48 33 +1
49 33 +1
42 18 +1
19 18 +1
18 27 +1
41 27 +1
53 27 +1
56 33 +1
56  8 +1
53  8 +1
41 18 +1
18 18 +1
23 27 +1
45 27 +1
56 27 +1
57  6 +1
55  8 +1
52  8 +1
40 18 +1
17 18 +1
27 27 +1
48 27 +1
56  6 +1
55  6 +1
54  6 +1
51  8 +1
39 18 +1
16 18 +1
29 27 +1
49 27 +1
53  6 +1
52  6 +1
51  6 +1
50  6 +1
38 18 +1
15 18 +1
30 29 +1
42 12 +1
41 12 +1
40 12 +1
39 12 +1
38 12 +1
33 14 +1
 9 20 +1
20 12 +1
19 12 +1
18 12 +1
17 12 +1
16 12 +1
15 12 +1
 9 14 +1
 2 14 +1
 6 27 +1
13 33 +1
19 33 +1
24 33 +1
28 33 +1
30 33 +1
31 33 +1
14 18 +1
13 27 +1
37 27 +1
42 33 +1
46 33 +1
46  8 +1
42  8 +1
37  8 +1
13 18 +1
19 27 +1
42 27 +1
49  6 +1
48  8 +1
45  8 +1
41  8 +1
36  8 +1
12 18 +1
24 27 +1
46 27 +1
48  6 +1
47  6 +1
44  8 +1
40  8 +1
35  8 +1
11 18 +1
28 27 +1
46  6 +1
45  6 +1
44  6 +1
43  6 +1
39  8 +1
34  8 +1
10 18 +1
30 27 +1
42  6 +1
41  6 +1
40  6 +1
39  6 +1
38  6 +1
33  8 +1
 9 18 +1
31 27 +1
37  6 +1
36  6 +1
35  6 +1
34  6 +1
33  6 +1
32  6 +1
 8 18 +1
14 12 +1
13 12 +1
12 12 +1
11 12 +1
10 12 +1
 9 12 +1
 8 12 +1
 1 14 +1
 7 27 +1
14 33 +1
20 33 +1
25 33 +1
25  8 +1
20  8 +1
14  8 +1
 7  8 +1
14 27 +1
31  6 +1
30  8 +1
28  8 +1
24  8 +1
19  8 +1
13  8 +1
 6  8 +1
20 27 +1
30  6 +1
29  6 +1
27  8 +1
23  8 +1
18  8 +1
12  8 +1
 5  8 +1
25 27 +1
28  6 +1
27  6 +1
26  6 +1
22  8 +1
17  8 +1
11  8 +1
 4  8 +1
25  6 +1
24  6 +1
23  6 +1
22  6 +1
21  6 +1
16  8 +1
10  8 +1
 3  8 +1
20  6 +1
19  6 +1
18  6 +1
17  6 +1
16  6 +1
15  6 +1
 9  8 +1
 2  8 +1
14  6 +1
13  6 +1
12  6 +1
11  6 +1
10  6 +1
 9  6 +1
 8  6 +1
 1  8 +1
 7  6 +1
 6  6 +1
 5  6 +1
 4  6 +1
 3  6 +1
 2  6 +1
 1  6 +1
 0  6 +1
