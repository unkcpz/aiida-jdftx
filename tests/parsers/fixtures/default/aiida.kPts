    0  [ +0.0625000 +0.0625000 +0.0625000 ]  0.007812500
    1  [ +0.0625000 +0.0625000 +0.1875000 ]  0.023437500
    2  [ +0.0625000 +0.0625000 +0.3125000 ]  0.023437500
    3  [ +0.0625000 +0.0625000 +0.4375000 ]  0.023437500
    4  [ +0.0625000 +0.0625000 -0.4375000 ]  0.023437500
    5  [ +0.0625000 +0.0625000 -0.3125000 ]  0.023437500
    6  [ +0.0625000 +0.0625000 -0.1875000 ]  0.023437500
    7  [ +0.0625000 +0.0625000 -0.0625000 ]  0.023437500
    8  [ +0.0625000 +0.1875000 +0.1875000 ]  0.023437500
    9  [ +0.0625000 +0.1875000 +0.3125000 ]  0.046875000
   10  [ +0.0625000 +0.1875000 +0.4375000 ]  0.046875000
   11  [ +0.0625000 +0.1875000 -0.4375000 ]  0.046875000
   12  [ +0.0625000 +0.1875000 -0.3125000 ]  0.046875000
   13  [ +0.0625000 +0.1875000 -0.1875000 ]  0.046875000
   14  [ +0.0625000 +0.1875000 -0.0625000 ]  0.046875000
   15  [ +0.0625000 +0.3125000 +0.3125000 ]  0.023437500
   16  [ +0.0625000 +0.3125000 +0.4375000 ]  0.046875000
   17  [ +0.0625000 +0.3125000 -0.4375000 ]  0.046875000
   18  [ +0.0625000 +0.3125000 -0.3125000 ]  0.046875000
   19  [ +0.0625000 +0.3125000 -0.1875000 ]  0.046875000
   20  [ +0.0625000 +0.3125000 -0.0625000 ]  0.046875000
   21  [ +0.0625000 +0.4375000 +0.4375000 ]  0.023437500
   22  [ +0.0625000 +0.4375000 -0.4375000 ]  0.046875000
   23  [ +0.0625000 +0.4375000 -0.3125000 ]  0.046875000
   24  [ +0.0625000 +0.4375000 -0.1875000 ]  0.046875000
   25  [ +0.0625000 +0.4375000 -0.0625000 ]  0.046875000
   26  [ +0.0625000 -0.4375000 -0.4375000 ]  0.023437500
   27  [ +0.0625000 -0.4375000 -0.3125000 ]  0.046875000
   28  [ +0.0625000 -0.4375000 -0.1875000 ]  0.046875000
   29  [ +0.0625000 -0.3125000 -0.3125000 ]  0.023437500
   30  [ +0.0625000 -0.3125000 -0.1875000 ]  0.046875000
   31  [ +0.0625000 -0.1875000 -0.1875000 ]  0.023437500
   32  [ +0.1875000 +0.1875000 +0.1875000 ]  0.007812500
   33  [ +0.1875000 +0.1875000 +0.3125000 ]  0.023437500
   34  [ +0.1875000 +0.1875000 +0.4375000 ]  0.023437500
   35  [ +0.1875000 +0.1875000 -0.4375000 ]  0.023437500
   36  [ +0.1875000 +0.1875000 -0.3125000 ]  0.023437500
   37  [ +0.1875000 +0.1875000 -0.1875000 ]  0.023437500
   38  [ +0.1875000 +0.3125000 +0.3125000 ]  0.023437500
   39  [ +0.1875000 +0.3125000 +0.4375000 ]  0.046875000
   40  [ +0.1875000 +0.3125000 -0.4375000 ]  0.046875000
   41  [ +0.1875000 +0.3125000 -0.3125000 ]  0.046875000
   42  [ +0.1875000 +0.3125000 -0.1875000 ]  0.046875000
   43  [ +0.1875000 +0.4375000 +0.4375000 ]  0.023437500
   44  [ +0.1875000 +0.4375000 -0.4375000 ]  0.046875000
   45  [ +0.1875000 +0.4375000 -0.3125000 ]  0.046875000
   46  [ +0.1875000 +0.4375000 -0.1875000 ]  0.046875000
   47  [ +0.1875000 -0.4375000 -0.4375000 ]  0.023437500
   48  [ +0.1875000 -0.4375000 -0.3125000 ]  0.046875000
   49  [ +0.1875000 -0.3125000 -0.3125000 ]  0.023437500
   50  [ +0.3125000 +0.3125000 +0.3125000 ]  0.007812500
   51  [ +0.3125000 +0.3125000 +0.4375000 ]  0.023437500
   52  [ +0.3125000 +0.3125000 -0.4375000 ]  0.023437500
   53  [ +0.3125000 +0.3125000 -0.3125000 ]  0.023437500
   54  [ +0.3125000 +0.4375000 +0.4375000 ]  0.023437500
   55  [ +0.3125000 +0.4375000 -0.4375000 ]  0.046875000
   56  [ +0.3125000 +0.4375000 -0.3125000 ]  0.046875000
   57  [ +0.3125000 -0.4375000 -0.4375000 ]  0.023437500
   58  [ +0.4375000 +0.4375000 +0.4375000 ]  0.007812500
   59  [ +0.4375000 +0.4375000 -0.4375000 ]  0.023437500
