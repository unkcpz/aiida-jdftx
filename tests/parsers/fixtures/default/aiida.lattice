lattice \
	   5.130606059000000    5.130606059000000    0.000000000000000  \
	   5.130606059000000    0.000000000000000    5.130606059000000  \
	   0.000000000000000    5.130606059000000    5.130606059000000 #Note: latt-scale has been absorbed into these lattice vectors.
