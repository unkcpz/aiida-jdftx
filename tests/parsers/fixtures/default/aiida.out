
*************** JDFTx 1.6.0 (git hash e9a0d98) ***************

Start date and time: Tue May 18 16:37:37 2021
Executable jdftx with command-line: -i aiida.in
Running on hosts (process indices):  DellArch (0-1)
Divided in process groups (process indices):  0 (0)  1 (1)
Resource initialization completed at t[s]:      0.00
Run totals: 2 processes, 2 threads, 0 GPUs


Input parsed successfully to the following command list (including defaults):

basis kpoint-dependent
coords-type Cartesian
core-overlap-check vector
coulomb-interaction Periodic
davidson-band-ratio 1.1
dump End IonicPositions Lattice ElecDensity Ecomponents Kpoints
dump-name aiida.$VAR
elec-cutoff 20 100
elec-eigen-algo Davidson
elec-ex-corr gga-PBE
electronic-minimize  \
	dirUpdateScheme      FletcherReeves \
	linminMethod         DirUpdateRecommended \
	nIterations          100 \
	history              15 \
	knormThreshold       0 \
	energyDiffThreshold  1e-08 \
	nEnergyDiff          2 \
	alphaTstart          1 \
	alphaTmin            1e-10 \
	updateTestStepSize   yes \
	alphaTreduceFactor   0.1 \
	alphaTincreaseFactor 3 \
	nAlphaAdjustMax      3 \
	wolfeEnergy          0.0001 \
	wolfeGradient        0.9 \
	fdTest               no
exchange-regularization WignerSeitzTruncated
fluid None
fluid-ex-corr (null) lda-PZ
fluid-gummel-loop 10 1.000000e-05
fluid-minimize  \
	dirUpdateScheme      PolakRibiere \
	linminMethod         DirUpdateRecommended \
	nIterations          100 \
	history              15 \
	knormThreshold       0 \
	energyDiffThreshold  0 \
	nEnergyDiff          2 \
	alphaTstart          1 \
	alphaTmin            1e-10 \
	updateTestStepSize   yes \
	alphaTreduceFactor   0.1 \
	alphaTincreaseFactor 3 \
	nAlphaAdjustMax      3 \
	wolfeEnergy          0.0001 \
	wolfeGradient        0.9 \
	fdTest               no
fluid-solvent H2O 55.338 ScalarEOS \
	epsBulk 78.4 \
	pMol 0.92466 \
	epsInf 1.77 \
	Pvap 1.06736e-10 \
	sigmaBulk 4.62e-05 \
	Rvdw 2.61727 \
	Res 1.42 \
	tauNuc 343133 \
	poleEl 15 7 1
forces-output-coords Positions
ion Si   0.000000000000000   0.000000000000000   0.000000000000000 0
ion Si   2.565303029500000   2.565303029500000   2.565303029500000 0
ion-species ./pseudo/si.upf
ion-width 0
ionic-minimize  \
	dirUpdateScheme      L-BFGS \
	linminMethod         DirUpdateRecommended \
	nIterations          0 \
	history              15 \
	knormThreshold       0.0001 \
	energyDiffThreshold  1e-06 \
	nEnergyDiff          2 \
	alphaTstart          1 \
	alphaTmin            1e-10 \
	updateTestStepSize   yes \
	alphaTreduceFactor   0.1 \
	alphaTincreaseFactor 3 \
	nAlphaAdjustMax      3 \
	wolfeEnergy          0.0001 \
	wolfeGradient        0.9 \
	fdTest               no
kpoint   0.500000000000   0.500000000000   0.500000000000  1.00000000000000
kpoint-folding 8 8 8 
latt-move-scale 1 1 1
latt-scale 1 1 1 
lattice  \
	   5.130606059000000    5.130606059000000    0.000000000000000  \
	   5.130606059000000    0.000000000000000    5.130606059000000  \
	   0.000000000000000    5.130606059000000    5.130606059000000 
lattice-minimize  \
	dirUpdateScheme      L-BFGS \
	linminMethod         DirUpdateRecommended \
	nIterations          0 \
	history              15 \
	knormThreshold       0 \
	energyDiffThreshold  1e-06 \
	nEnergyDiff          2 \
	alphaTstart          1 \
	alphaTmin            1e-10 \
	updateTestStepSize   yes \
	alphaTreduceFactor   0.1 \
	alphaTincreaseFactor 3 \
	nAlphaAdjustMax      3 \
	wolfeEnergy          0.0001 \
	wolfeGradient        0.9 \
	fdTest               no
lcao-params -1 1e-06 0.001
pcm-variant GLSSA13
spintype no-spin
subspace-rotation-factor 1 yes
symmetries automatic
symmetry-threshold 0.0001



---------- Setting up symmetries ----------

Found 48 point-group symmetries of the bravais lattice
Found 48 space-group symmetries with basis
Applied RMS atom displacement 0 bohrs to make symmetries exact.

---------- Initializing the Grid ----------
R = 
[      5.13061      5.13061            0  ]
[      5.13061            0      5.13061  ]
[            0      5.13061      5.13061  ]
unit cell volume = 270.107
G =
[   0.612324   0.612324  -0.612324  ]
[   0.612324  -0.612324   0.612324  ]
[  -0.612324   0.612324   0.612324  ]
Minimum fftbox size, Smin = [  36  36  36  ]
Chosen fftbox size, S = [  36  36  36  ]

---------- Initializing tighter grid for wavefunction operations ----------
R = 
[      5.13061      5.13061            0  ]
[      5.13061            0      5.13061  ]
[            0      5.13061      5.13061  ]
unit cell volume = 270.107
G =
[   0.612324   0.612324  -0.612324  ]
[   0.612324  -0.612324   0.612324  ]
[  -0.612324   0.612324   0.612324  ]
Minimum fftbox size, Smin = [  32  32  32  ]
Chosen fftbox size, S = [  32  32  32  ]

---------- Exchange Correlation functional ----------
Initalized PBE GGA exchange.
Initalized PBE GGA correlation.

---------- Setting up pseudopotentials ----------
Width of ionic core gaussian charges (only for fluid interactions / plotting) set to 0

Reading pseudopotential file './pseudo/si.upf':
  'Si' pseudopotential, 'PBE' functional
  Generated using ONCVPSP code by D. R. Hamann
  Author: Martin Schlipf and Francois Gygi  Date: 150915.
  4 valence electrons, 2 orbitals, 4 projectors, 1510 radial grid points, with lMax = 1
  Transforming local potential to a uniform radial grid of dG=0.02 with 1833 points.
  Transforming nonlocal projectors to a uniform radial grid of dG=0.02 with 432 points.
    3S    l: 0   occupation:  2.0   eigenvalue: -0.397365
    3P    l: 1   occupation:  2.0   eigenvalue: -0.149981
  Transforming atomic orbitals to a uniform radial grid of dG=0.02 with 432 points.
  Core radius for overlap checks: 2.98 bohrs.

Initialized 1 species with 2 total atoms.

Folded 1 k-points by 8x8x8 to 512 k-points.

---------- Setting up k-points, bands, fillings ----------

WARNING: k-mesh symmetries are a subgroup of size 12
The effectively sampled k-mesh is a superset of the specified one,
and the answers need not match those with symmetries turned off.
Reduced to 60 k-points under symmetry. 
Computing the number of bands and number of electrons
Calculating initial fillings.
nElectrons:   8.000000   nBands: 4   nStates: 60

----- Setting up reduced wavefunction bases (one per k-point) -----
average nbasis = 1153.992 , ideal nbasis = 1153.918

---------- Setting up ewald sum ----------
Optimum gaussian width for ewald sums = 2.330232 bohr.
Real space sum over 1331 unit cells with max indices [  5  5  5  ]
Reciprocal space sum over 2197 terms with max indices [  6  6  6  ]

---------- Allocating electronic variables ----------
Initializing wave functions:  linear combination of atomic orbitals
Si pseudo-atom occupations:   s ( 2 )  p ( 2 )
	FillingsUpdate:  mu: +0.279616392  nElectrons: 8.000000
LCAOMinimize: Iter:   0  Etot: -7.8208612407387559  |grad|_K:  6.417e-05  alpha:  1.000e+00
	FillingsUpdate:  mu: +0.279525933  nElectrons: 8.000000
LCAOMinimize: Iter:   1  Etot: -7.8208707770944894  |grad|_K:  1.455e-06  alpha:  9.531e-01  linmin: -2.158e-03  cgtest:  5.119e-03  t[s]:      1.74
	FillingsUpdate:  mu: +0.279527405  nElectrons: 8.000000
LCAOMinimize: Iter:   2  Etot: -7.8208707822287167  |grad|_K:  7.895e-09  alpha:  9.983e-01  linmin: -5.767e-03  cgtest: -1.891e-03  t[s]:      2.15
	FillingsUpdate:  mu: +0.279527395  nElectrons: 8.000000
LCAOMinimize: Iter:   3  Etot: -7.8208707822288552  |grad|_K:  2.133e-10  alpha:  9.390e-01  linmin: -3.340e-01  cgtest:  6.850e-01  t[s]:      2.55
LCAOMinimize: Encountered beta<0, resetting CG.
LCAOMinimize: Converged (|Delta Etot|<1.000000e-06 for 2 iters).


---- Citations for features of the code used in this run ----

   Software package:
      R. Sundararaman, K. Letchworth-Weaver, K.A. Schwarz, D. Gunceler, Y. Ozhabes and T.A. Arias, 'JDFTx: software for joint density-functional theory', SoftwareX 6, 278 (2017)

   gga-PBE exchange-correlation functional:
      J.P. Perdew, K. Burke and M. Ernzerhof, Phys. Rev. Lett. 77, 3865 (1996)

   Total energy minimization:
      T.A. Arias, M.C. Payne and J.D. Joannopoulos, Phys. Rev. Lett. 69, 1077 (1992)

This list may not be complete. Please suggest additional citations or
report any other bugs at https://github.com/shankar1729/jdftx/issues

Initialization completed successfully at t[s]:      2.60


-------- Electronic minimization -----------
ElecMinimize: Iter:   0  Etot: -7.820870782228857  |grad|_K:  3.343e-04  alpha:  1.000e+00
ElecMinimize: Iter:   1  Etot: -7.871513555343051  |grad|_K:  1.482e-04  alpha:  1.635e+00  linmin: -7.002e-04  t[s]:      3.21
ElecMinimize: Iter:   2  Etot: -7.879841820749974  |grad|_K:  8.042e-05  alpha:  1.367e+00  linmin: -6.119e-05  t[s]:      3.56
ElecMinimize: Iter:   3  Etot: -7.882441478456930  |grad|_K:  3.374e-05  alpha:  1.450e+00  linmin: -1.899e-04  t[s]:      3.91
ElecMinimize: Iter:   4  Etot: -7.882808178485556  |grad|_K:  1.563e-05  alpha:  1.162e+00  linmin: -7.745e-05  t[s]:      4.25
ElecMinimize: Iter:   5  Etot: -7.882900303957758  |grad|_K:  7.199e-06  alpha:  1.361e+00  linmin: -8.536e-06  t[s]:      4.60
ElecMinimize: Iter:   6  Etot: -7.882920992203719  |grad|_K:  4.719e-06  alpha:  1.440e+00  linmin: -4.034e-06  t[s]:      4.95
ElecMinimize: Iter:   7  Etot: -7.882931632695081  |grad|_K:  2.856e-06  alpha:  1.724e+00  linmin:  8.215e-06  t[s]:      5.30
ElecMinimize: Iter:   8  Etot: -7.882934972737859  |grad|_K:  1.696e-06  alpha:  1.478e+00  linmin:  1.058e-07  t[s]:      5.65
ElecMinimize: Iter:   9  Etot: -7.882936179708237  |grad|_K:  9.898e-07  alpha:  1.514e+00  linmin: -1.349e-07  t[s]:      6.00
ElecMinimize: Iter:  10  Etot: -7.882936650856108  |grad|_K:  6.172e-07  alpha:  1.735e+00  linmin: -2.318e-07  t[s]:      6.35
ElecMinimize: Iter:  11  Etot: -7.882936799116714  |grad|_K:  3.147e-07  alpha:  1.404e+00  linmin:  1.322e-08  t[s]:      6.69
ElecMinimize: Iter:  12  Etot: -7.882936843936966  |grad|_K:  1.873e-07  alpha:  1.633e+00  linmin:  1.284e-08  t[s]:      7.04
ElecMinimize: Iter:  13  Etot: -7.882936860476034  |grad|_K:  1.243e-07  alpha:  1.701e+00  linmin:  1.960e-08  t[s]:      7.39
ElecMinimize: Iter:  14  Etot: -7.882936867765558  |grad|_K:  7.246e-08  alpha:  1.703e+00  linmin: -7.738e-07  t[s]:      7.73
ElecMinimize: Iter:  15  Etot: -7.882936870129939  |grad|_K:  4.383e-08  alpha:  1.625e+00  linmin: -2.078e-06  t[s]:      8.08
ElecMinimize: Converged (|Delta Etot|<1.000000e-08 for 2 iters).
Setting wave functions to eigenvectors of Hamiltonian

# Ionic positions in cartesian coordinates:
ion Si   0.000000000000000   0.000000000000000   0.000000000000000 0
ion Si   2.565303029500000   2.565303029500000   2.565303029500000 0

# Forces in Cartesian coordinates:
force Si   0.000000000000000  -0.000000000000000   0.000000000000000 0
force Si   0.000000000000000  -0.000000000000000   0.000000000000000 0

# Energy components:
   Eewald =       -8.3994724711985196
       EH =        0.5503370049656557
     Eloc =       -2.5565693402334086
      Enl =        1.8451129095899654
      Exc =       -2.4078586928891821
       KE =        3.0855137196355500
-------------------------------------
     Etot =       -7.8829368701299387

IonicMinimize: Iter:   0  Etot: -7.882936870129939  |grad|_K:  0.000e+00  t[s]:      8.14
IonicMinimize: Converged (|grad|_K<1.000000e-04).

#--- Lowdin population analysis ---
# oxidation-state Si +0.037 +0.037


Dumping 'aiida.ionpos' ... done
Dumping 'aiida.lattice' ... done
Dumping 'aiida.n' ... done
Dumping 'aiida.Ecomponents' ... done
Dumping 'aiida.kPts' ... done
Dumping 'aiida.kMap' ... done
End date and time: Tue May 18 16:37:45 2021  (Duration: 0-0:00:08.17)
Done!
